terraform-modules/storage-bucket/bucket.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
```

Comment:
this should go in the versions.tf file.  do you need to specify a patch version or is it ok just to be >= 0.12.  versions.tf says >= 0.12.  if 0.12.6 is min supported then update versions.tf with that.  still remove it from here.

I realize that you copied this from another module but I think it is confusing to have these spread out in diff parts of module.  the entire module should just specify the min version any resource it creates needs.